src/main/resources/application.yml

```
@@ -1,3 +1,4 @@
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)